.drone.star

```diff
@@ -36,6 +36,17 @@ PYTHON = "python"
 THEGEEKLAB_DRONE_GITHUB_COMMENT = "thegeeklab/drone-github-comment:1"
 TOOLHIPPIE_CALENS = "toolhippie/calens:latest"
 
+secrets = {
```

Comment:
What is this indirection good for? To have an overview of the used secrets? There should be a comment that explains why this is needed.

Reply:
Well, if a secret changes, you delete it in the backend and create a new one of the same name, don't you?

Reply:
No, see current changes...